From e1b9303fdac3241235b793bd450e3472587abaaa Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:17 -0400
Subject: [redhat] kabi: Adding symbol set_page_dirty (mm/page-writeback.c)

Message-id: <14e570d8740b885ed4aca4e0fd54f661a501b079.1410510773.git.poros@redhat.com>
Patchwork-id: 96009
O-Subject: [PATCH RHEL7.1 197/233] kabi: Adding symbol set_page_dirty (mm/page-writeback.c)
Bugzilla: 1071099
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Used to mark the page as dirty in write operations.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/set_page_dirty
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/set_page_dirty
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/set_page_dirty

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/set_page_dirty b/redhat/kabi/kabi-module/kabi_ppc64/set_page_dirty
new file mode 100644
index 0000000..6089b81
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/set_page_dirty
@@ -0,0 +1,2 @@
+#1-
+0xe4fc15c4 set_page_dirty vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/set_page_dirty b/redhat/kabi/kabi-module/kabi_s390x/set_page_dirty
new file mode 100644
index 0000000..99228bc
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/set_page_dirty
@@ -0,0 +1,2 @@
+#1-
+0x25c8152e set_page_dirty vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/set_page_dirty b/redhat/kabi/kabi-module/kabi_x86_64/set_page_dirty
new file mode 100644
index 0000000..4b59eaf
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/set_page_dirty
@@ -0,0 +1,2 @@
+#1-
+0x120eee66 set_page_dirty vmlinux EXPORT_SYMBOL
-- 
1.7.1